recipyCommon/version_control.py

```diff
@@ -80,7 +80,6 @@ def add_svn_info(run, scriptpath):
         run["svncommit"] = svn_info["commit_revision"]
         if not option_set('ignored metadata', 'diff'):
             run['diff'] = svn_diff(svn_info["wc-info/wcroot-abspath"])
-    except (SvnException, ValueError):
+    except:
```

Comment:
What exception was causing the problem here? Ideally we should have a list of exceptions to catch, so that we don't catch unwanted ones by mistake (this was happening once before in the `get_version` code).

At the very least, we should use `except Exception:` which means it won't catch things that don't inherit from `Exception` (so it won't catch `KeyboardInterrupt` or other exceptions that are used to stop the program running and shouldn't be caught internally).